CODEOWNERS

```diff
@@ -1 +1 @@
-* @sigstore/core-team
+* @sigstore/core-team @sigstore/community-codeowners
```

Comment:
Error will resolve once PR is merged, since community-owners will have write perms added.